### PR TITLE
Trivial improvements to Makefile.menhir.

### PR DESCRIPTION
**Pull request by fpottier**
One fix for Linux/MacOS portability.
Extra verbosity for comfort.

### CODE REVIEW

File: Makefile.menhir [hunk 1]
```diff
@@ -47,8 +47,9 @@ promote-menhir: parsing/parser.mly
 .PHONY: test-menhir
 test-menhir: parsing/parser.mly
 	$(MENHIR) $(MENHIRFLAGS) parsing/parser.mly
-	sed -i "s/MenhirLib/CamlinternalMenhirLib/g" \
-	  $(addprefix parsing/parser.,ml mli)
+	sed -e "s/MenhirLib/CamlinternalMenhirLib/g" -i.bak \
+	  $(addprefix parsing/parser.,ml mli) \
+	  && rm parsing/parser.*.bak
```
Comment (gasche) [inline Makefile.menhir]: I shouldn't have used `-i` in the first place, but I have no idea what `-i.bak` is and I don't want to hear about it. Would you be patient enough to just use `cat | sed > foo.tmp; mv foo.tmp foo`?

**Comment by fpottier** [comment]
@gasche: updated as per your request.

**Comment by gasche** [comment]
The new trunk is more recent than what you used as it includes my Makefile.menhir documentation commit. Could you rebase against it? (And maybe include your "included" typo fix in your PR?)

**Comment by fpottier** [comment]
Done.

**Review by gasche — APPROVED** [review]
Good to go once CI passes. Thanks!

**Comment by gasche** [comment]
In fact this PR touches code that is never run by the CI. I checked manually that `make promote-menhir` still works, and I am merging now.

@fpottier: life tip, when you decide to prepare a git PR, you should always create a fresh branch (`git checkout -b trivial-makefile-menhir-changes`) and push it to your git clone when you want to create the PR. This lets you work on several unmerged PRs in parallel, and update each of them on the github interface by using `git push --force` on their branch.

**Pull request merged**